corehq/apps/locations/management/commands/set_location_id.py

```diff
@@ -35,7 +35,7 @@ def _is_location_domain(domain):
                 return domain_cache[domain]
             else:
                 domain_obj = Domain.get_by_name(domain)
-                val = domain_obj.commtrack_enabled or domain_obj.locations_enabled
+                val = domain_obj.commtrack_enabled or domain_obj.uses_locations
```

Comment:
Actually `commtrack_enabled` is a prerequisite of `uses_locations`, so this is redundant.  I'll update.